pymmcore_plus/_tests/test_core.py

```diff
@@ -55,6 +56,34 @@ def test_search_paths(core: CMMCorePlus):
         core.setDeviceAdapterSearchPaths("test_path")
 
 
+def test_load_system_config(core: CMMCorePlus):
+    with pytest.raises(FileNotFoundError):
+        core.loadSystemConfiguration("nonexistent")
+
+    test_dir = Path(__file__).parent.absolute()
+    orig_dir = Path.cwd()
+    os.chdir(test_dir)
```

Comment:
will this work without using `chdir`, just passing `test_dir /  "local_config.cfg"` to loadSystemConfig?

Reply:
I think it would, except that you can in theory run these tests from any directory - so for that to pass you are assuming that you're running them from the repo top level.

Reply:
the `test_dir` object is a pathlib.Path object, and line 63 makes it absolutely relative to this file (independent of the current working dir)... so it should work